cloud/pkg/cloudhub/servers/httpserver/nodetask/checknodename.go

```diff
@@ -0,0 +1,43 @@
+package nodetask
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/emicklei/go-restful"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog/v2"
+
+	"github.com/kubeedge/kubeedge/cloud/pkg/common/client"
+)
+
+func CheckNodeName(request *restful.Request, response *restful.Response) {
+	// get nodeName
+	nodeName := request.QueryParameter("nodeName")
```

Comment:
You could consider using path parameters `request.PathParameter(..)`.

Reply:
Thank you for your suggestion, but my API is designed as `/nodenamecheck?nodeName=%s`, so `request.PathParameter(..)`is not appropriate here.

Reply:
Yes, would it be more RESTful to use path parameters?

Reply:
Ok, thank you for your suggestion. I have made corresponding modifications